app/controllers/admin/topical_event_featurings_controller.rb

```diff
@@ -1,36 +1,30 @@
 class Admin::TopicalEventFeaturingsController < Admin::BaseController
   before_action :load_topical_event
   before_action :load_topical_event_featuring, only: %i[confirm_destroy destroy]
-  layout :get_layout
+  layout "design_system"
 
   def index
     filter_params = params.slice(:page, :type, :author, :organisation, :title)
                           .permit!
                           .to_h
-                          .merge(state: "published", topical_event: @topical_event.to_param)
-
-    filter_params = filter_params.merge(per_page: Admin::EditionFilter::GOVUK_DESIGN_SYSTEM_PER_PAGE) if preview_design_system?(next_release: false)
+                          .merge(
+                            state: "published",
+                            topical_event: @topical_event.to_param,
+                            per_page: Admin::EditionFilter::GOVUK_DESIGN_SYSTEM_PER_PAGE,
+                          )
 
     @filter = Admin::EditionFilter.new(Edition, current_user, filter_params)
     @tagged_editions = editions_to_show
 
     @topical_event_featurings = @topical_event.topical_event_featurings
     @featurable_offsite_links = @topical_event.offsite_links
-
-    if request.xhr?
```

Comment:
Should the XHR request response functionality have been removed?

Reply:
We're not longer making async calls when searching which is what this handles.